src/dataexplorer/dataexplorer-application/src/components/Navigation/Navigation.tsx

```diff
@@ -1,83 +1,98 @@
-import React from 'react';
-import { Menu, Layout } from 'antd';
+import React, { useState } from 'react';
+import { Layout, Avatar } from 'antd';
+import Breadcrumbs from './Breadcrumbs/Breadcrumbs';
 import 'antd/dist/antd.css';
-import Dashboard from '../Dashboard/Dashboard';
-import MySchemas from '../MySchemas/MySchemas';
-import MyPatients from '../MyPatients/MyPatients';
-import { Switch, Route, Link, withRouter } from 'react-router-dom';
-import MenuItem from 'antd/lib/menu/MenuItem';
+import MyPatients from '../PatientInfo/MyPatients';
+import { Switch, Route, withRouter, Link } from 'react-router-dom';
+import { UserOutlined } from '@ant-design/icons';
+import PatientInfo from '../PatientInfo/PatientInfo';
+import { History } from 'history';
 
-const { Header, Content, Sider, Footer } = Layout;
-const data = {
-    items: [
-        {
-            id: 1,
-            url: '/',
-            title: 'Hjem',
-        },
-        {
-            id: 2,
-            url: '/Skjema',
-            title: 'Mine Skjema',
-        },
-        {
-            id: 3,
-            url: '/Pasient',
-            title: 'Mine pasienter',
-        },
-        {
-            id: 4,
-            url: '/Search',
-            title: 'Filtrer på egenskaper',
-        },
-    ],
-};
+interface IHistory {
+    history: History;
+}
+
+const { Header, Content } = Layout;
+const Navigation = (props: IHistory) => {
```

Comment:
This interface is a duplicate of another one created in 'types/IHistory'.
Since this only takes one prop I think it would work to drop both IHistory interfaces and just do like
```suggestion
const { Header, Content } = Layout;
const Navigation = (props: History) => {
```